src/workerd/server/workerd.capnp

```diff
@@ -72,6 +72,13 @@ struct Config {
   # WARNING: Use at your own risk. V8 flags can have all sorts of wild effects including completely
   #   breaking everything. V8 flags also generally do not come with any guarantee of stability
   #   between V8 versions. Most users should not set any V8 flags.
+
+  builtins @3 :List(BuiltinsBundle);
```

Comment:
Why group the modules into "bundles"? Could this just be `List(BuiltinModule)`? (I don't feel strongly but the extra layer looks redundant to me.)

Reply:
The idea is to have pre-produced bundles with group of modules for a given api. E.g. node-compat bundle, or some-new-service bundle. This way clients need to specify a bundle, and not read and concatenate module lists from different bundles. Also it makes it possible to change bundles without changing the way you reference them.

Reply:
Do you think we need `struct SharedWorkerConfig` though? I am not 100% comfortable passing server config deep into worker intialization.

Reply:
> The idea is to have pre-produced bundles with group of modules

OK, fair enough.

> Do you think we need struct SharedWorkerConfig though? I am not 100% comfortable passing server config deep into worker intialization.

`SharedWorkerConfig` seems too generic. What if we add a new setting that needs to be passed into _some other_ deep part of the implementation?

I could imagine we wrap the builtins in something called `BuiltinModuleConfig`, which is passed to the specific places it is needed. Or we could just pass down `capnp::List<BuiltinsBundle>::Reader`.

I'm not actually too worried about it though.

Side note: I wonder if we should rename "builtins" to "extensions"? Feels like a more natural term for what this does.

Reply:
> I wonder if we should rename "builtins" to "extensions"? Feels like a more natural term for what this does.

done.